src/codemagic/tools/_app_store_connect/abstract_base_action.py

```diff
@@ -55,6 +57,9 @@ def create_beta_build_localization(
             should_print: bool = True) -> BetaBuildLocalization:
         ...
 
+    def add_build_to_beta_groups(self, build_id: Union[ResourceId, Build], beta_group_names: Sequence[str]):
```

Comment:
Argument name `build_id` is misleading as the value can also be full `Build` object. To be more consistent with already existing naming please consider using `build` instead.
```suggestion
    def add_build_to_beta_groups(self, build: Union[ResourceId, Build], beta_group_names: Sequence[str]):
```

Reply:
Signature is not updated in `AbstractBaseAction`.